tests/settings/SettingTest.py

```diff
@@ -100,6 +100,43 @@ def test_typed_list(self):
 
         self.assertEqual(repr(typed_list(int)), 'typed_list(int)')
 
+    def test_typed_list_shortcuts(self):
+        # int
+        self.uut = Setting('key', '1, 2, 3')
+        self.assertEqual(int_list(self.uut), [1, 2, 3])
+
+        with self.assertRaises(ValueError):
+            self.uut = Setting('key', '1, a, 3')
+            int_list(self.uut)
+
+        self.assertEqual(repr(int_list), 'typed_list(int)')
+
+        # str
+        self.uut = Setting('key', 'a, b, c')
+        self.assertEqual(str_list(self.uut), ['a', 'b', 'c'])
+
+        self.assertEqual(repr(str_list), 'typed_list(str)')
+
+        # float
+        self.uut = Setting('key', '0.8, 1.3, 5.87')
+        self.assertEqual(float_list(self.uut), [0.8, 1.3, 5.87])
+
+        with self.assertRaises(ValueError):
+            self.uut = Setting('key', '1.987, a, 3')
+            float_list(self.uut)
+
+        self.assertEqual(repr(float_list), 'typed_list(float)')
+
+        # bool
+        self.uut = Setting('key', 'true, nope, yeah')
+        self.assertEqual(bool_list(self.uut), [True, False, True])
+
+        with self.assertRaises(ValueError):
+            self.uut = Setting('key', 'true, false, 78, 89.0')
+            bool_list(self.uut)
+
+        self.assertEqual(repr(bool_list), 'typed_list(bool)')
```

Comment:
you should move each subtests for your specialized types into an own test function. E.g. `test_str_list`, `test_int_list` etc.
Using separate test functions makes test design more modular :+1: